include/Yash.h

```diff
@@ -175,127 +117,122 @@ class Yash {
             return;
         default:
             if (m_ctrlState == CtrlState::LeftBracket) {
-                m_ctrlSeq += character;
-                for (const auto &it : m_supportedCtrlSeq) {
-                    if (m_ctrlSeq.compare(0, m_ctrlSeq.length(), it.first, 0, m_ctrlSeq.length()) == 0) {
-                        if (m_ctrlSeq.length() == it.first.length()) {
-                            switch (it.second) {
-                            case CtrlSeq::Up:
-                                if (m_commandsIndex != m_commands.begin()) {
-                                    m_command = *--m_commandsIndex;
+                m_ctrlCharacter += character;
+                for (uint8_t index = 0; index < s_ctrlCharacters.size(); ++index) {
+                    if (m_ctrlCharacter.compare(0, m_ctrlCharacter.length(), s_ctrlCharacters[index], 0, m_ctrlCharacter.length()) == 0) {
+                        if (m_ctrlCharacter.length() == s_ctrlCharacters[index].length()) {
+                            switch (index) {
```

Comment:
Should we have a default statement -since it checks on a unsigned int?

Reply:
Yes - of cause. Not sure why the compiler did not warn about this